packages/superset-ui-number-format/src/NumberFormats.ts

```diff
@@ -3,6 +3,10 @@ const DOLLAR_SIGNED = '+$,.2f';
 const DOLLAR_ROUND = '$,d';
 const DOLLAR_ROUND_SIGNED = '+$,d';
 
+const DURATION = 'DURATION';
```

Comment:
Please revert the changes to this file. The duration formatter should be optional and not registered by default. There are other apps using this package and some do not need duration formatter. You can register these formatters in `incubator-superset`'s `setupFormatter.js`.  That way other apps can take advantage of tree-shaking and won't have to bundle `pretty-ms`.